scripts/build.py

```diff
@@ -144,8 +144,13 @@
     dest='minify_third_party_libs_only')
 
 
-def generate_app_yaml():
-    """Generate app.yaml from app_dev.yaml."""
+def generate_app_yaml(is_prod_mode=False):
+    """Generate app.yaml from app_dev.yaml.
+
+    Args:
+        is_prod_mode: bool. Whether the script is being called
+        with --prod_env flag or not.
```

Comment:
This should be indented by 2 I think....

Reply:
I just verified the indentation is correct.